docs/SSO.rst

```diff
@@ -1767,39 +1760,13 @@ eXo Platform as SAML2 SP
    
```

Comment:
>  .. note:: Add the option **--no-compat** for tomcat application server.

=>

Delete note and add **--no-compat** in installation command line

Reply:
Since the new packaging is not yet ready, this option is still mandatory to install exo-saml in tomcat